styleguide/components/styleguide/styleguide.css

```diff
@@ -1,28 +1,23 @@
 @import "../../../src/styles/colors-v2.css";
+@import "../../../src/styles/spacing.css";
 
 .root {
-  background-color: var(--white);
-  margin-top: 65px;
-  padding-left: 200px;
+  display: grid;
+  grid-template-columns: max-content auto;
+  grid-template-rows: auto;
+  grid-template-areas:
+    "header header "
+    "sidebar main ";
 }
 
-.content {
-  max-width: 1000px;
-  padding: 16px 32px;
-  margin: 0 auto;
-
-  /* For IE11 */
-  display: block;
+.sidebar {
```

Comment:
We should also somehow make the width "sticky", so that when items are filtered out, the width doesn't shrink.

Reply:
> We should probably still set a max-width for the sidebar,

No, the sidebar's width is set by the grid which fits its contents. 

> maybe have it be scrollable on the x axis?

I thought about this and decided against this. 

1. The main content should fit the viewport assuming a reasonable size -- and thus now scrolling.
2. If the viewport is tiny enough that there is horizontal scrolling (think cellphone type of deal), we probably want to allow the user to see most of the content without a sticky navigation. Also, we probably want to consider a responsive design (in the future). 

> We should also somehow make the width "sticky", so that when items are filtered out, the width doesn't shrink.

Oh, nice find! I didn't know there was a bug. I can add a min-width.

Reply:
@juanca I was more worried about long component names and the indentation of sub-groups causing the sidebar nav to grow to take up too much space from the content. As it is right now, it feels too wide to me, but I'm fine with punting this for a future re-design.

Reply:
Okay, I actually don't think we can do anything about it. 

- The input's width is set by the grid column
- The grid column's width is set its content (and thus the navigation items)

There isn't a way to tell the input to keep a static width without setting a static width. 
The input is created by Styleguidist which means we can't customize it by using some `useLayoutEffect` hook. 

I think perhaps the best way around this is to change the search behavior to be more like a "highlight the list items" or "scroll to matching item" type of deal. Could be interesting. And this is aligned with my point below about how we need a DRD for this functionality.

Reply:
Yeah, if we're going to go with this, I think it'd actually be better to just nuke the search and rely on browser native finding functionality.